test/conformance/README.md

```diff
@@ -9,7 +9,7 @@ To update the list, run
 
 ```console
 bazel build //test/conformance:list_conformance_tests
-cp bazel-genfiles/test/conformance/conformance.yaml test/conformance/testdata
+cp bazel-bin/test/conformance/conformance.yaml test/conformance/testdata
```

Comment:
confirmed this change is valid by using bazel-2.2.0 like
```
$ bazel-2.2.0 build //test/conformance:list_conformance_tests
Extracting Bazel installation...
Starting local Bazel server and connecting to it...
... still trying to connect to local Bazel server after 10 seconds ...
WARNING: Download from https://storage.googleapis.com/k8s-bazel-cache/https://github.com/bazelbuild/rules_go/releases/download/v0.22.2/rules_go-v0.22.2.tar.gz failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException GET returned 404 Not Found
WARNING: Download from https://storage.googleapis.com/k8s-bazel-cache/https://github.com/kubernetes/repo-infra/archive/v0.0.3.tar.gz failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException GET returned 404 Not Found
INFO: Analyzed target //test/conformance:list_conformance_tests (1833 packages loaded, 17271 targets configured).
INFO: Found 1 target...
Target //test/conformance:list_conformance_tests up-to-date:
  bazel-bin/test/conformance/conformance.yaml
INFO: Elapsed time: 930.104s, Critical Path: 54.24s
INFO: 2881 processes: 2881 linux-sandbox.
INFO: Build completed successfully, 2885 total action
$ ls bazel-genfiles/test/conformance/conformance.yaml
ls: cannot access 'bazel-genfiles/test/conformance/conformance.yaml': No such file or directory
$ ls bazel-bin/test/conformance/conformance.yaml
bazel-bin/test/conformance/conformance.yaml
$
```
So looks good to me.
/lgtm